Dockerfile

```diff
@@ -31,21 +31,23 @@ RUN curl -sSL https://raw.githubusercontent.com/python-poetry/poetry/master/get-
 ENV PATH "/root/.poetry/bin:/opt/venv/bin:${PATH}"
 
 # copy files
-COPY . /app/
+COPY . /build/
+
+# change working directory
+WORKDIR /build
 
 # install dependencies
 RUN python -m venv /opt/venv && \
   . /opt/venv/bin/activate && \
   pip install --no-cache-dir -U 'pip<20' && \
-  cd /app && \
-  poetry install --no-dev --no-interaction
+  poetry install --no-dev --no-root --no-interaction && \
+  poetry run pip install .
```

Comment:
This step wasn't here before? Do we need it?

Reply:
I mean `poetry run pip install`

Reply:
Yes!
The idea is poetry by default installs your app in editable mode (when you just run `poetry install`). By doing `poetry install --no-root` we install everything except `rasa` itself, and `poetry run pip install .` installs rasa in non-editable mode using the already installed dependencies.

Reply:
There are other ways of doing the same (for example, https://stackoverflow.com/questions/53835198/integrating-python-poetry-with-docker) but they all do more or less the same in the end.

Reply:
I think it should be `poetry pip install --no-deps` (https://github.com/python-poetry/poetry/issues/1382#issuecomment-588544067)

Reply:
One minute, will check it out

Reply:
looks like that.. @alwx do you know if pip would use the poetry lockfile?

Reply:
I'd guess that no